__tests__/app.test.js

```diff
@@ -163,6 +163,61 @@ describe("/api/articles", () => {
           });
         });
     });
+    // testing for sort and order options - status 200
+    const sortOptions = [
+      "article_id",
+      "title",
+      "topic",
+      "author",
+      "created_at",
+      "votes",
+    ];
+    const orderOptions = ["asc", "desc"];
+    const jestOptionObj = {
+      asc: { descending: false },
+      desc: { descending: true },
+    };
+
+    for (const orderOption of orderOptions) {
+      for (const sortOption of sortOptions) {
+        test(`status 200 for ?sort_by=${sortOption}&order=${orderOption} returns the articles in the article table sorted as queried by users`, () => {
+          return request(app)
+            .get(`/api/articles?sort_by=${sortOption}&order=${orderOption}`)
+            .expect(200)
+            .then(({ body: { articles } }) => {
+              expect(articles.length).toBe(13);
+              expect(articles).toBeSortedBy(
+                sortOption,
+                jestOptionObj[orderOption]
+              );
+            });
+        });
+      }
+    }
```

Comment:
Testing using loops is not something I would consider necessary when doing integration testing, it's not about testing every possible combination of queries, it's about testing each query being requested and responding with what you would expect. With integration testing, one test of `?sort_by` and one test of `?order_by` is enough to show good confidence in the queries